src/managing-workflow/production-deployments.md

```diff
@@ -1,3 +1,57 @@
 # Production Deployments
 
-TODO (bacongobbler): rewrite for v2
+When readying a Workflow deployment for production workloads, there are some additional (but
+optional) recommendations.
+
+
+## Running Workflow without Minio
+
+Workflow makes use of [Minio][] to provide persistent storage for the [Registry][], [Database][],
+and [Logger][] components. The additional operational complexity of Minio is tolerated because of
+the need for persistent storage for platform high availability.
+
+Alternatively, persistent storage can be achieved by running an external S3-compatible blob store,
```

Comment:
Really this should be encouraged rather than presented as just an alternative. Maybe this could just remove `Alternatively,`?